signer/src/context/signer_state.rs

```diff
@@ -207,6 +217,16 @@ impl SbtcLimits {
     pub fn max_mintable_cap(&self) -> Amount {
         self.max_mintable_cap.unwrap_or(Amount::MAX_MONEY)
     }
+
+    /// Get the number of blocks over which the rolling withdrawal cap is applied.
+    pub fn rolling_withdrawal_blocks(&self) -> u64 {
+        self.rolling_withdrawal_blocks.unwrap_or(0)
```

Comment:
Maybe 1 would be a safer default? A 0-width rolling window may be ill defined. Also in `unlimited` below

Reply:
Talked offline, the rationale is that we want consistent inputs, and if don't have them we want to stop processing withdrawals; so:
 - Emily should validate that you either set both or none of them when updating the limits, and fail if only one of them is set;
 - The signers will stop processing withdrawals (and log it) if they only have one of them (eg, set with manual intervention on db);

So I guess for this PR we want to remove the unwrap for both of them and return the "raw" value.
Then in https://github.com/stacks-network/sbtc/pull/1515 we will add the signer side logic. And I guess in a different PR add the Emily validation.